torchvision/models/efficientnet.py

```diff
@@ -779,7 +779,9 @@ def efficientnet_b0(
     weights = EfficientNet_B0_Weights.verify(weights)
 
     inverted_residual_setting, last_channel = _efficientnet_conf("efficientnet_b0", width_mult=1.0, depth_mult=1.0)
-    return _efficientnet(inverted_residual_setting, 0.2, last_channel, weights, progress, **kwargs)
+    return _efficientnet(
+        inverted_residual_setting, kwargs.pop("dropout", 0.2), last_channel, weights, progress, **kwargs
```

Comment:
Is `pop`'ing safe here or should we just access, i.e.

```suggestion
        inverted_residual_setting, kwargs.get("dropout", 0.2), last_channel, weights, progress, **kwargs
```

?

Reply:
Pop is safe and needs to happen. Otherwise kwargs will provide it twice.